dev-docs/design/how-lit-html-works.md

```diff
@@ -258,38 +253,37 @@ Usually text-position markers will be comment nodes, but inside `<style>`, `<scr
 
 ### 2.ii. Create
 
-The create phase is performed only when rendering for **the first time to a specific Part**.
+The create phase is performed only when rendering for **the first time to a specific container's ChildPart**.
 
 In the `lit-html` source code this phase is the [instantiating and cloning](https://github.com/lit/lit/blob/64fb960246e8f1eae982c2f09fca9759001756af/packages/lit-html/src/lit-html.ts#L1534-L1535) of a `TemplateInstance`.
 
 For the example, this phase results in the DOM nodes being cloned into a `fragment`, ready to be inserted in the document, and a list of `Parts` that can be updated by the next phase to set the dynamic JavaScript values in that fragment.
 
 ```js
 const instance = new TemplateInstance(preparedTemplate);
+// The created DOM fragment will be inserted into the DOM.
 const fragment = instance._clone();
 
 console.log(instance._$parts);
-/*
-[AttributePart, ChildPart, EventPart]
-*/
+// [AttributePart, ChildPart, EventPart]
 ```
 
 After the update phase this fragment is inserted into the DOM.
 
 The following sections cover in detail what happens when the `TemplateInstance` is created and then cloned:
 
-#### Create a `TemplateInstance`
+#### Create a `TemplateInstance` and instantate `Part`s
 
 A `TemplateInstance` is responsible for creating the initial DOM and updating that DOM. It's an updatable instance of a `Template`. The `TemplateInstance` holds references to the `Part`s used to update the DOM.
 
-#### Clone the template and instantiate Parts
-
 Within `TemplateInstance._clone()`, first the `<template>` element is cloned into a document fragment.
 
 After cloning, the document fragment node tree is walked to associate nodes with `TemplatePart`s by depth-first-index. When a node's index matches the index stored on a `TemplatePart`, a `Part` instance is created for the node. Based on the data in the `TemplatePart`, one of `ChildPart`, `AttributePart`, `PropertyPart`, `EventPart`, `BooleanAttributePart`, or `ElementPart` is instantiated.
 
 These Part instances are stored on the `TemplateInstance`.
 
+The `TemplateInstance`, with its `Part`s, is then stored on the `ChildPart` so that future render requests can use it to skip directly to the update phase.
```

Comment:
I think I omit `container's` because the ChildPart containing the `TemplateInstance` is not always linked to a container.

For example:
``render(document.body, html`<p>${html`<span>Hi</span>`}</p>`);``

Here `document.body` ends up with a direct litPart property referring to the root `ChildPart`. This is what I think of when referring to `container's ChildPart`. But then the attached `TemplateInstance` containing the `<p>` element will have a `ChildPart` in its parts array, and that ChildPart contains the `TemplateInstance` with the `<span>`.

I guess ChildParts do keep a reference to the start/end nodes in which they occupy in the DOM tree. WDYT about adding `container's` vs something else to clarify that distinction?

Reply:
That's valid. It does then also call into question line 256 here calling it container's `ChildPart`: https://github.com/lit/lit/blob/f4f4ecbfed3aa2c9ffb07a3bfb33557624e88236/dev-docs/design/how-lit-html-works.md?plain=1#L256

Container here doesn't necessarily have to refer to a Node but could refer to the `TemplateInstance` that is holding the ChildPart to render the TemplateResult into. Other alternatives might be "hosting ChildPart" or "target ChildPart"?

Reply:
I just paged back in the context of this part of the doc.

In this specific place in the document, the `TemplateInstance` is stored directly on the ChildPart, not on the owner of the ChildPart (be it the container element or a parent TemplateInstance). It is stored on the `_$committedValue` field here: https://github.com/lit/lit/blob/how-lit-html-works/packages/lit-html/src/lit-html.ts#L1556

Reply:
Yeah I'm trying to see if it's possible to describe that `ChildPart`. Because here in this part of the doc we just talked about a `TemplateInstance` was made and many `Part`s, which could also include a `ChildPart` among other types, were made and stored on the instance. And then to say this `TemplateInstance` is stored on the `ChildPart`, I feel like that `ChildPart` needs more description. We called it the container's `ChildPart` at the top of the "Create" section. If not container's, I was suggesting either host or target `ChildPart`.

Reply:
per our discussion
```suggestion
The `TemplateInstance`, with its `Part`s, is then stored on the root `ChildPart` so that future render requests can use it to skip directly to the update phase.
```

Reply:
Love it!